dq-game/src/server/controller.py

```diff
@@ -140,15 +140,18 @@ def clear_current_answers(self):
     def ask_question(self, question, players, answer_limit):
 
         blocked_players_for_wrong_answers = [
-            player.name for player in players if player.blocked_for_wrong_answer
+            player for player in players if player.blocked_for_wrong_answer
         ]
 
         blocked_players_by_other_players = [
-            player.name for player in players if player.blocked_by
+            player for player in players if player.blocked_by
         ]
 
         for blocked_player in blocked_players_by_other_players:
-            self.socket.send_to_socket_named(blocked_player, "BLOCKED", "event")
+            self.socket.send_to_socket_named(
+                blocked_player.name, blocked_player.blocked_by, "data-blocking-player"
+            )
+            self.socket.send_to_socket_named(blocked_player.name, "BLOCKED", "event")
```

Comment:
Did you try to use only the socket event ? 
`self.socket.send_to_socket_named({ to: blocked_player.name, by: blocked_player.blocked_by}, "BLOCKED", "event")`

Reply:
I would need to refactor several pieces of code to do this. I would not change it